<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Safety</class>
 <widget class="QWidget" name="Safety">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Safety Lock</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Analog Input (AIn8) Safe Range (V)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Analog_Enable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Analog_Lower">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Analog_Upper">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="Digital_Enable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Digital Input Safety</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Digital_Channels">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>202</width>
      <height>267</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Channel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alarm when</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>25</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>1</string>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>26</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>27</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>28</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>29</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>30</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>31</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>32</string>
     </property>
     <property name="flags">
      <set>ItemIsEnabled</set>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
